datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -1124,6 +1124,11 @@ SELECT COUNT(*) FROM aggregate_test_100
 ----
 100
 
+query I
```

Comment:
Can you please also add a negative test (aka a test that shows an unknown table)?

Perhaps something like

```sql
query error
SELECT COUNT(foo.*) FROM aggregate_test_100
```

Reply:
I'll add this test case late, thanks!